src/stylesheets/core/_functions.scss

```diff
@@ -978,14 +946,14 @@ system orders
 
 /*
 ----------------------------------------
-spacing()
+units()
 ----------------------------------------
 Converts a spacing unit multiple into
```

Comment:
I know you didn't write this comment but feels unclear. Should it be multiplier?

Reply:
It definitely works like a multiplier, but since we've been using `spacing units` as the term for our standard whitespace palette, it started to make more sense to use `units()` — I think I'm open to other names for this function, but since this is one of the primary user-facing functions (like `color()`) I'd like a name that works to connects the function to the concept.

Reply:
Oh, and I did write that comment, so I can accept full responsibility!

Reply:
Yeah, sorry, I meant the word `multiple` should be replaced with `multiplier`. The name of the function is totally clear